recipes/syft/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+go_compiler_version:
```

Comment:
Please delete this file, the default should suffice if you use the different macro below